grammars/c.gbnf

```diff
@@ -0,0 +1,42 @@
+root ::= (declaration)*
+
+declaration ::= dataType identifier "(" parameter? ")" "{" statement* "}"
+
+dataType  ::= "int" ws | "float" ws | "char" ws
+identifier ::= [a-zA-Z_] [a-zA-Z_0-9]*
+
+parameter ::= dataType identifier
+
+statement ::=
+    ( dataType identifier ws "=" ws expression ";" ) |
```

Comment:
suggestion: I adopted the convention of always specifying whitespace after literal strings or character ranges (when allowed/required). This makes it easier to keep track of. In this example, `ws` could be moved to the end of `identifier` (and assumed wherever `identifier` is used), while the semicolon should allow optional trailing whitespace.